docs/topics/testing/tools.txt

```diff
@@ -235,7 +235,12 @@ Use the ``django.test.Client`` class to make requests.
         file-processing code expects.)
 
         You may also provide any file-like object (e.g., :class:`~io.StringIO` or
-        :class:`~io.BytesIO`) as a file handle.
+        :class:`~io.BytesIO`) as a file handle but in order to satisfy
+        :data:`~django.core.validators.validate_image_file_extension` validator
```

Comment:
In the midst of discussion of general file upload, suddenly there's a mention of something that's  specific to `ImageField`. I think the phrasing could be improved.